api/test/integration/throttle-guard.e2e-spec.ts

```diff
@@ -0,0 +1,38 @@
+import { Test, TestingModule } from '@nestjs/testing';
+import { INestApplication } from '@nestjs/common';
+import request from 'supertest';
+import { AppModule } from '../../src/modules/app.module';
+import { PrismaService } from '../../src/services/prisma.service';
+
+describe('Throttle Guard Tests', () => {
```

Comment:
What endpoint still uses this?

Reply:
This is a bit unfortunate, but while writing this test out I found out that we were double calling the guard. Once on every request and once through specific useGuard() calls

part of this pr pulled all those useGuards out so we just have the general one that was already in place